syntax = "proto3";
package cdm;
option java_package = "com.kitware.physiology.cdm";

enum eSide
{
  NullSide = 0;
  Left     = 1;
  Right    = 2;
}

enum eGate
{
  NullGate = 0;
  Open     = 1;
  Closed   = 2;
}

enum eSwitch
{
  NullSwitch = 0;
  Off        = 1;
  On         = 2;
}

enum eCharge
{
  NullCharge = 0;/** @brief Not provided */
  Negative   = 1;
  Neutral    = 2;
  Positive   = 3;
}

enum eDecimalFormatType
{
  SystemFormatting  = 0;/*<<@brief Not provided, let the system do what it wants */
  DefaultFloat      = 1;
  FixedMantissa     = 2;
  SignificantDigits = 3;
}


enum eDataRequestCategory
{
  Patient            = 0;
  Physiology         = 1;
  Environment        = 2;
  GasCompartment     = 3;
  LiquidCompartment  = 4;
  ThermalCompartment = 5;
  TissueCompartment  = 6;
  Substance          = 7;
  AnesthesiaMachine  = 8;
  ECG                = 9;
  Inhaler            = 10;
}

